cmd/operator/run.go

```diff
@@ -68,44 +59,41 @@ func runProtoform(configPath string) {
 	}
 
 	// Log Kubernetes version
-	kversion, err := bdutil.GetKubernetesVersion(deployer.KubeClientSet)
+	kversion, err := util.GetKubernetesVersion(deployer.KubeClientSet)
 	if err == nil {
-		logrus.Infof("Kubernetes: %s", kversion)
+		log.Infof("Kubernetes: %s", kversion)
 	}
 
 	// Log Openshift version
-	oversion, err := bdutil.GetOcVersion(deployer.KubeClientSet)
+	oversion, err := util.GetOcVersion(deployer.KubeClientSet)
 	if err == nil {
-		logrus.Infof("Openshift: %s", oversion)
+		log.Infof("Openshift: %s", oversion)
 	}
 
 	stopCh := make(chan struct{})
 
-	//sampleController := sample.NewCRDInstaller(deployer.Config, deployer.KubeConfig, deployer.KubeClientSet, bdutil.GetSampleDefaultValue(), stopCh)
-	//deployer.AddController(sampleController)
-
-	alertController := alert.NewCRDInstaller(deployer.Config, deployer.KubeConfig, deployer.KubeClientSet, bdutil.GetAlertTemplate(), stopCh)
-	deployer.AddController(alertController)
-
-	hubController := blackduck.NewCRDInstaller(deployer.Config, deployer.KubeConfig, deployer.KubeClientSet, bdutil.GetBlackDuckTemplate(), stopCh)
-	deployer.AddController(hubController)
-
-	opssSightController, err := opssight.NewCRDInstaller(&opssight.Config{
-		Config:        deployer.Config,
-		KubeConfig:    deployer.KubeConfig,
-		KubeClientSet: deployer.KubeClientSet,
-		Defaults:      bdutil.GetOpsSightDefault(),
-		Threadiness:   deployer.Config.Threadiness,
-		StopCh:        stopCh,
-	})
-	if err != nil {
-		panic(err)
+	// get the operator scope from an environment variable
+	var isClusterScoped bool
+	clusterScopeEnv, ok := os.LookupEnv("CLUSTER_SCOPE")
```

Comment:
Everything else in the operator is configured using viper. Should CLUSTER_SCOPE and CRD_NAMES be configured the same way?

Reply:
done